client/public/clientJS/partnerCarousel.js

```diff
@@ -0,0 +1,67 @@
+$(document).ready(function () {
+  'use strict';
+  var smallParent3 = $('.small3'),
+    smallUlLi3 = $('.small3 .small-ul li'),
+    smallSpan3 = $('.small3 .small-span'),
+    smallSetLeft3,
+    smallRunLeft3;
+  // Reset Li length
+  smallUlLi3.parent().attr('data-small3', smallUlLi3.length);
+  // Run Set
+  smallSetLeft3 = function () {
+    var smallUlLi3 = $('.small3 .small-ul li');
```

Comment:
double declaration?

Reply:
Not sure what you mean by double declaration. Can you explain please?

Reply:
```var smallUlLi3 = $('.small3 .small-ul li');``` is already declared in the global scope of you function